backend/typescript/rest/campRoutes.ts

```diff
@@ -0,0 +1,19 @@
+import { Router } from "express";
+import ICampService from "../services/interfaces/campService";
+import CampService from "../services/implementations/campService";
+import { getErrorMessage } from "../utilities/errorUtils";
+
+const campRouter: Router = Router();
+
+const campService: ICampService = new CampService();
+
+campRouter.get("/csv/:id", async (req, res) => {
+  try {
+    const csvString = await campService.generateCampersCSV(req.params.id);
+    res.status(200).json(csvString);
```

Comment:
We might have to set the content type to 
```suggestion
    res.status(200).set('Content-Type', 'text/csv').send(csvString);
```

Reply:
Good call, and sure, I'll update this!